### ISSUE TITLE
Use own dataset

### ISSUE THREAD

**Issue by YSYYU** [zju3dv/DetectorFreeSfM#37]
May I ask if I can use my own dataset? If so, how can I modify it

**Comment by hxy-123** [comment]
Yes, you can first try out our demo code. If everything runs properly, you can format your own data same with demo data, and run our system.
